src/test/java/org/folio/atom/feed/reader/AtomFeedReaderApplicationTest.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.atom.feed.reader;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import jakarta.validation.Valid;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.web.bind.annotation.RestController;
+
+import org.folio.spring.liquibase.FolioLiquibaseConfiguration;
+import org.folio.tenant.domain.dto.TenantAttributes;
+import org.folio.tenant.rest.resource.TenantApi;
+
+@ActiveProfiles({"test", "testcontainers-pg"})
```

Comment:
Could we use @EnablePostgres and @IntegrationTest from the folio testing library instead of PostgresTestContainersBeanFactoryPostProcessor under 'testcontainers-pg' profile?

Reply:
Removed tests as the application doesn't have any logic